extras/create_send_notifications.py

```diff
@@ -106,10 +118,6 @@ def main():
     # Change to the correct output directory
     os.chdir(os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR))
 
-    # Build list of CyHy orgs
-    cyhy_org_ids = build_cyhy_org_list(db)
-    logging.debug("Found {} CYHY orgs: {}".format(len(cyhy_org_ids), cyhy_org_ids))
-
```

Comment:
This is where you should be calling `build_notifications_org_list()` and logging a debug message to output the list of orgs that it found.

Don't forget to that whatever variable you choose in place of `cyhy_org_ids` (e.g. `notification_org_ids`) should replace  `cyhy_org_ids` in the remaining lines of this script.

Reply:
Please see https://github.com/cisagov/cyhy-reports/pull/96/commits/e38846eab2fbfad3e8d6d8affe6555101c9928b1

Reply:
Looks like everything in this conversation has now been addressed.